gallery.php

```diff
@@ -335,7 +335,7 @@
   </div>
 </body>
 <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
-<script src="http://netdna.bootstrapcdn.com/bootstrap/3.0.2/js/bootstrap.min.js"></script>
+<!--<script src="http://netdna.bootstrapcdn.com/bootstrap/3.0.2/js/bootstrap.min.js"></script> -->
```

Comment:
@vinaychandil Did commenting out this line actually do the trick? Is bootstrap included somewhere else in the same file?